src/bin/main.rs

```diff
@@ -74,6 +79,12 @@ fn parse_args_and_run() -> Result<(), Error> {
     if opt.cli {
         meta.is_cli = true;
     }
+    if opt.name != None {
```

Comment:
We can call `.is_some()` here (which is idiomatic), instead of doing a comparison. :smile: